tests/Support/TestCase.php

```diff
@@ -43,6 +44,6 @@ public function createRequest(array $body = []): ServerRequest
 
     public function createRequestModelFactory(ContainerInterface $container): RequestModelFactory
     {
-        return new RequestModelFactory(new Injector($container));
+        return new RequestModelFactory(new Validator(null), new Injector($container));
```

Comment:
How about to use mock of `ValidatorInterface`?

Reply:
Why?